### DISCUSSION TITLE
Download Bulk or Folder

### DISCUSSION THREAD

**Discussion by vioarief** [BetaHuhn/spaces-cli#230] [Ideas]
Hai, 

Is it possible to download as a bulk or like download folder so we can use it for backup case ?

**Comment by BetaHuhn** [comment]
No, downloading a whole folder is not supported.

If you know which files to download you could go through each one and download it individually, but you can't specify a folder to download as a whole. You can only upload an entire directory.

Feel free to create a PR to add this functionality if you need it.
